### ISSUE TITLE
Fints PSD2 support

### ISSUE THREAD

**Issue by varac** [MoritzR/fints2ledger#3]
And I wonder why the current requirements are still `fints>=2,<3`, doesn't it work with latest fints ?
I'm asking because [fints has added PSD2 support](https://github.com/raphaelm/python-fints/commit/95079bc522a2905febe485cbdc2ed3d45c557c87) in v. [0.3.0](https://github.com/raphaelm/python-fints/releases/tag/v3.0.0).
Would be great to use PSD2 with fints2ledger.

**Comment by varac** [comment]
To reproduce, using a standard fints2ledger 0.5.2 installation I get this:

```
❯ fints2ledger --ledger-file ~/.config/fints2ledger/test.ledger
You should register your program with the ZKA and pass your own product_id as a parameter.
Dialog response: 9050 - Die Nachricht enthält Fehler.
Dialog response: 9075 - Starke Kundenauthentifizierung notwendig.
Traceback (most recent call last):
  File "/home/varac/.local/bin/fints2ledger", line 8, in <module>
    sys.exit(main())
  File "/home/varac/.local/pipx/venvs/fints2ledger/lib/python3.7/site-packages/fints2ledger/main.py", line 9, in main
    Fints2Csv(config).retrieveAndSave()
  File "/home/varac/.local/pipx/venvs/fints2ledger/lib/python3.7/site-packages/fints2ledger/fints2csv.py", line 24, in retrieveAndSave
    transaction), retriever.get_hbci_transactions(self.config["fints"]["start"], Date.today())))
  File "/home/varac/.local/pipx/venvs/fints2ledger/lib/python3.7/site-packages/fints2ledger/transaction_retriever.py", line 7, in get_hbci_transactions
    accounts = self.client.get_sepa_accounts()
  File "/home/varac/.local/pipx/venvs/fints2ledger/lib/python3.7/site-packages/fints/client.py", line 433, in get_sepa_accounts
    for seg in response.find_segments(HISPA1, throw=True):
  File "/home/varac/.local/pipx/venvs/fints2ledger/lib/python3.7/site-packages/fints/types.py", line 293, in find_segments
    'The bank\'s response did not contain a response to your request, please inspect debug log.'
fints.exceptions.FinTSNoResponseError: The bank's response did not contain a response to your request, please inspect debug log.
```

Using `pip install git+https://github.com/raphaelm/python-fints.git@refs/pull/95/head` to install [PSD2 support](https://github.com/raphaelm/python-fints/pull/95) I at least get around the `Dialog response: 9075 - Starke Kundenauthentifizierung notwendig.` error (but bump into a different issue).

**Comment by MoritzR** [comment]
There is a new version available (0.5.3 - now also marked with a git tag), which upgrades to pyfints 3 and should fix the issue

**Comment by varac** [comment]
Look like this one is fixed, thx!
